### PR TITLE
UG-846272: volume 3 features UG

### PR DESCRIPTION
**Pull request by Sri-hari-haran-g**
Hi,

I have added UG Documentation for the below samples,
•	Title:  Inject Custom CSS
•	Title: Inject Custom JavaScript
•	Title: Performance Optimization.
•	Title: Support to get the Image Rotation angle from OCR Processor

Regards,
Srihariharan Gopalakrishnan

### CODE REVIEW

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 1]
```diff
@@ -1267,4 +1267,86 @@ End Using
 
 {% endhighlight %}
 
+{% endtabs %}  
+
+new
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-OCR/Features.md:1272]: Remove unwanted lines

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 2]
```diff
@@ -1834,3 +1834,171 @@ document.Close(True)
 {% endhighlight %}
 
 {% endtabs %}
+
+## Inject Custom CSS
+
+The Blink rendering engine provides the support for inject the custom CSS to be applied in HTML or URL before rendering to PDF.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1840]: Use this - "The Blink rendering engine supports the injection of custom CSS to be applied to HTML or a URL before rendering it into a PDF document using the 'Css' property of the 'BlinkConverterSettings' class."

Comment (SowmiyaLoganathan) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:1840]: Provide API link for BlinkConverterSettings class

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 3]
```diff
@@ -1834,3 +1834,171 @@ document.Close(True)
...
+The Blink rendering engine provides the support for inject the custom CSS to be applied in HTML or URL before rendering to PDF. 
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Initialize HTML to PDF converter.
+HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+BlinkConverterSettings blinkConverterSettings = new BlinkConverterSettings();
+
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1849]: Remove unwanted spaces

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 4]
```diff
@@ -1834,3 +1834,171 @@ document.Close(True)
...
+document.Save(fileStream)
+document.Close(true)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Inject Custom JavaScript
+
+The Blink rendering engine provides the support for inject the custom JavaScript to be applied in HTML or URL before rendering to PDF.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1899]: Use this - "The Blink rendering engine offers support for injecting custom JavaScript to be applied to HTML or a URL before rendering it into a PDF document using the 'JavaScript' property of the 'BlinkConverterSettings' class."

Comment (SowmiyaLoganathan) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:1899]: Provide API link for BlinkConverterSettings class

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 5]
```diff
@@ -1834,3 +1834,171 @@ document.Close(True)
...
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Initialize HTML to PDF converter.
+HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+BlinkConverterSettings blinkConverterSettings = new BlinkConverterSettings();
+
+//Set Blink viewport size.
+blinkConverterSettings.ViewPortSize = new Size(1280, 0);
+blinkConverterSettings.Margin.All = 30;
+
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1912]: Remove unwanted spaces

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 6]
```diff
@@ -1834,3 +1834,171 @@ document.Close(True)
...
+document.Save(fileStream)
+document.Close(true)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Performance Optimization.
+
+The Blink rendering engine provides support for Reuse the Browser Process to optimize the HTML to PDF performance of multiple operations.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1959]: The Blink rendering engine provides support for reusing the browser process to optimize the HTML to PDF performance for multiple operations using the 'ReuseBrowserProcess' property of the 'HtmlToPdfConverter' class.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 7]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+document.Save(fileStream)
+document.Close(true)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Performance Optimization.
+
+The Blink rendering engine provides support for reusing the browser process to optimize the HTML to PDF performance for multiple operations using the 'ReuseBrowserProcess' property of the 'HtmlToPdfConverter' class.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1941]: Add link for HtmlToPdfConverter class

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 8]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+Dim fileStream As FileStream = New FileStream("Test.pdf", FileMode.CreateNew, FileAccess.ReadWrite)
+'Save and close the PDF document.
+document.Save(fileStream)
+document.Close(true)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Inject Custom JavaScript
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1889]: Change the title as "Inject custom JavaScript"

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 9]
```diff
@@ -1267,4 +1267,84 @@ End Using
 
 {% endhighlight %}
 
+{% endtabs %}  
+
+## Support to get the Image Rotation angle from OCR Processor
+
+For Getting the Image rotation angle, you can rotate the image with 4 angles (0,90,180,360) from OCR Processor. This feature is working in multiple Images and multiple pages. 
+
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-OCR/Features.md:1275]: Remove the space between sentence

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 10]
```diff
@@ -1267,4 +1267,82 @@ End Using
 
 {% endhighlight %}
 
+{% endtabs %}  
+
+## Get image rotation angle from OCR processor
+
+For Getting the Image rotation angle, you can rotate the image with 4 angles (0,90,180,360) from OCR Processor. This feature is working in multiple Images and multiple pages. The following code sample illustrates to support to get the Image Rotation angle from OCR Processor:
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-OCR/Features.md:1274]: Add full stop in end of sentence

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 11]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
 {% endhighlight %}
 
 {% endtabs %}
+
+## Inject custom CSS
+
+The Blink rendering engine supports the injection of custom CSS to be applied to HTML or a URL before rendering it into a PDF document using the 'Css' property of the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1840]: The Blink rendering engine supports the injection of custom CSS to be applied to the HTML or a URL before rendering it into a PDF document using the 'CSS' property of the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 12]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+
+## Inject custom CSS
+
+The Blink rendering engine supports the injection of custom CSS to be applied to HTML or a URL before rendering it into a PDF document using the 'Css' property of the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class 
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Initialize HTML to PDF converter.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1846]: //Initialize the HTML to PDF converter.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 13]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+BlinkConverterSettings blinkConverterSettings = new BlinkConverterSettings();
+//Set Blink viewport size.
+blinkConverterSettings.ViewPortSize = new Size(1280, 0);
+blinkConverterSettings.Margin.All = 30;
+//Set custom CSS
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1852]: //Set the custom CSS.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 14]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+blinkConverterSettings.Css = "body {\r\n background-color: red; \r\n}";
+htmlConverter.ConverterSettings = blinkConverterSettings;
+//Convert URL to PDF document.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1855]: //Convert the URL to a PDF document.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 15]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+The Blink rendering engine supports the injection of custom CSS to be applied to HTML or a URL before rendering it into a PDF document using the 'Css' property of the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class 
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Initialize HTML to PDF converter.
+HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+BlinkConverterSettings blinkConverterSettings = new BlinkConverterSettings();
+//Set Blink viewport size.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1849]: //Set the Blink viewport size.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 16]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+FileStream fileStream = new FileStream("Test.pdf", FileMode.CreateNew, FileAccess.ReadWrite);
+//Save and close the PDF document.
+document.Save(fileStream);
+document.Close(true);
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Initialize HTML to PDF converter.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1867]: 'Initialize the HTML to a PDF converter.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 17]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+Dim htmlConverter As HtmlToPdfConverter = New HtmlToPdfConverter()
+Dim blinkConverterSettings As BlinkConverterSettings = New BlinkConverterSettings()
+
+'Set Blink viewport size.
+blinkConverterSettings.ViewPortSize = New Size(1280, 0)
+blinkConverterSettings.Margin.All = 30
+'Set custom CSS
+blinkConverterSettings.Css = "body {\r\n background-color: red; \r\n}"
+htmlConverter.ConverterSettings = blinkConverterSettings
+'Convert URL to PDF document.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1877]: 'Convert the URL to a PDF document.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 18]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+document.Save(fileStream)
+document.Close(true)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Inject custom JavaScript
+
+The Blink rendering engine offers support for injecting custom JavaScript to be applied to HTML or a URL before rendering it into a PDF document using the 'JavaScript' property of the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1891]: The Blink rendering engine offers support for injecting custom JavaScript to be applied to the HTML or a URL before rendering it into a PDF document using the 'JavaScript' property of the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 19]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Initialize HTML to PDF converter.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1897]: //Initialize the HTML to a PDF converter.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 20]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+BlinkConverterSettings blinkConverterSettings = new BlinkConverterSettings();
+//Set Blink viewport size.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1900]: //Set the Blink viewport size.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 21]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+blinkConverterSettings.ViewPortSize = new Size(1280, 0);
+blinkConverterSettings.Margin.All = 30;
+//Set Custom JavaScript
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1903]: //Set the Custom JavaScript.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 22]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+blinkConverterSettings.JavaScript = "document.querySelectorAll('img').forEach((node)=>{node.remove();})";
+htmlConverter.ConverterSettings = blinkConverterSettings;
+//Convert URL to PDF document.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1906]: //Convert the URL to a PDF document.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 23]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+FileStream fileStream = new FileStream("Test.pdf", FileMode.CreateNew, FileAccess.ReadWrite);
+//Save and close the PDF document.
+document.Save(fileStream);
+document.Close(true);
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Initialize HTML to PDF converter.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1918]: 'Initialize the HTML to a PDF converter.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 24]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+Dim htmlConverter As HtmlToPdfConverter = New HtmlToPdfConverter()
+Dim blinkConverterSettings As BlinkConverterSettings = New BlinkConverterSettings()
+'Set Blink viewport size.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1921]: 'Set the Blink viewport size.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 25]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+blinkConverterSettings.ViewPortSize = new Size(1280, 0)
+blinkConverterSettings.Margin.All = 30
+'Set Custom JavaScript
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1924]: 'Set the Custom JavaScript.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 26]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+blinkConverterSettings.JavaScript = "document.querySelectorAll('img').forEach((node)=>{node.remove();})"
+htmlConverter.ConverterSettings = blinkConverterSettings
+'Convert URL to PDF document.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1927]: 'Convert the URL to a PDF document.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 27]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+document.Save(fileStream)
+document.Close(true)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Performance optimization
+
+The Blink rendering engine provides support for reusing the browser process to optimize the HTML to PDF performance for multiple operations using the 'ReuseBrowserProcess' property of the [HtmlToPdfConverter](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.HtmlToPdfConverter.html) class.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1941]: The Blink rendering engine provides support for reusing the browser process to optimize the HTML to a PDF performance for multiple operations using the 'ReuseBrowserProcess' property of the [HtmlToPdfConverter](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.HtmlToPdfConverter.html) class.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 28]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Initialize HTML to PDF converter.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1947]: //Initialize the HTML to a PDF converter.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 29]
```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
...
+HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+//Reuse the browser instance.           
+htmlConverter.ReuseBrowserProcess = true;
+for (int i = 0; i < 10; i++)        
+{           
+    BlinkConverterSettings settings = new BlinkConverterSettings();
+    settings.AdditionalDelay = 1000;
+    settings.EnableJavaScript = true;
+    htmlConverter.ConverterSettings = settings;        
+    //Convert URL to PDF document.
```
Comment (florenceliliana) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1957]: //Convert the URL to a PDF document.

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 30]
```diff
@@ -1267,4 +1267,82 @@ End Using
 
 {% endhighlight %}
 
+{% endtabs %}  
+
+## Get image rotation angle from OCR processor
+
+For Getting the Image rotation angle, you can rotate the image with 4 angles (0,90,180,360) from OCR Processor. This feature is working in multiple Images and multiple pages. The following code sample illustrates to support to get the Image Rotation angle from OCR Processor.
```
Comment (florenceliliana) [inline File-Formats/PDF/Working-with-OCR/Features.md:1274]: To get the Image rotation angle, you can rotate the image with 4 angles (0,90,180, and 360) from the OCR Processor. This feature works in multiple Images and multiple pages. The following code sample illustrates support for the Image Rotation angle from the OCR Processor.

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 31]
```diff
@@ -1267,4 +1267,82 @@ End Using
...
+
+{% tabs %}  
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Initialize the OCR processor.@
+using (OCRProcessor processor = new OCRProcessor())
+{
+    //Get stream from an image file.
```
Comment (florenceliliana) [inline File-Formats/PDF/Working-with-OCR/Features.md:1283]: //Get the stream from an image file.

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 32]
```diff
@@ -1267,4 +1267,82 @@ End Using
...
+{% tabs %}  
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Initialize the OCR processor.@
+using (OCRProcessor processor = new OCRProcessor())
+{
+    //Get stream from an image file. 
+    FileStream stream = new FileStream(@"D:\Input.pdf", FileMode.Open);
+    //Set OCR language to process.
```
Comment (florenceliliana) [inline File-Formats/PDF/Working-with-OCR/Features.md:1285]: //Set the OCR language to process.

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 33]
```diff
@@ -1267,4 +1267,82 @@ End Using
...
+    PdfLoadedDocument document = new PdfLoadedDocument(stream);
+    //Set OCR language.
```
Comment (florenceliliana) [inline File-Formats/PDF/Working-with-OCR/Features.md:1287]: //Set the OCR language.

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 34]
```diff
@@ -1267,4 +1267,82 @@ End Using
...
+    processor.Settings.Language = Languages.English;
+    //Sets Unicode font to preserve the Unicode characters in a PDF document.
```
Comment (florenceliliana) [inline File-Formats/PDF/Working-with-OCR/Features.md:1289]: //Set the Unicode font to preserve the Unicode characters in a PDF document.

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 35]
```diff
@@ -1267,4 +1267,82 @@ End Using
...
+}
+
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Initialize the OCR processor
+Using processor As OCRProcessor = New OCRProcessor()
+    'Get stream from an image file.
```
Comment (florenceliliana) [inline File-Formats/PDF/Working-with-OCR/Features.md:1319]: 'Get the stream from an image file.

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 36]
```diff
@@ -1267,4 +1267,82 @@ End Using
...
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Initialize the OCR processor
+Using processor As OCRProcessor = New OCRProcessor()
+    'Get stream from an image file. 
+    Dim stream As FileStream = New FileStream(@"D:\Input.pdf", FileMode.Open);
+    'Set OCR language to process.
```
Comment (florenceliliana) [inline File-Formats/PDF/Working-with-OCR/Features.md:1321]: 'Set the OCR language to process.

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 37]
```diff
@@ -1267,4 +1267,82 @@ End Using
...
+    Dim document As PdfLoadedDocument = New PdfLoadedDocument(stream);
+    'Set OCR language.
```
Comment (florenceliliana) [inline File-Formats/PDF/Working-with-OCR/Features.md:1323]: 'Set the OCR language.

File: File-Formats/PDF/Working-with-OCR/Features.md [hunk 38]
```diff
@@ -1267,4 +1267,82 @@ End Using
...
+    processor.Settings.Language = Languages.English;
+    'Sets Unicode font to preserve the Unicode characters in a PDF document.
```
Comment (florenceliliana) [inline File-Formats/PDF/Working-with-OCR/Features.md:1325]: 'Set the Unicode font to preserve the Unicode characters in a PDF document.

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The content review is completed.

**Review by florenceliliana — APPROVED** [review]
Approved.

**Review by SowmiyaLoganathan — APPROVED** [review]

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 39]
```diff
@@ -1834,3 +1834,152 @@ document.Close(True)
 {% endhighlight %}
 
 {% endtabs %}
+
+## Inject custom CSS
+
+The Blink rendering engine supports the injection of custom CSS to be applied to HTML or a URL before rendering it into a PDF document using the 'CSS' property of the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class
```
Comment (PrakashViswanath) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1840]: 'CSS'  to  'Css'

**Review by PrakashViswanath — CHANGES_REQUESTED** [review]

**Review by PrakashViswanath — APPROVED** [review]
The changes are fine.

**Pull request merged**